models/VerificationGateway/models/VerificationGateway/VerificationGatewayDomain/VerificationGateway/VerificationGateway.masl

```diff
@@ -11,14 +11,18 @@ theEventDetails : string;
 theAuditEventData : string;
```

Comment:
General question: I've assumed that it is expected behaviour for log messages at **different** log levels to be recorded in any given log?

Reply:
I am not sure I understand the question... partly because "log level" can mean more than one thing.

1) Log levels (e.g. Debug, Information, Warning, etc) can be used to decide the destination of log messages.
2) Log hierarchy (e.g. pv.sv, pv.sv.happyjob, etc) can be used to influence the destination of the log messages.
3) We can mix and match and log in more than one place.

Reply:
I used the wrong term in my question - sorry.
I meant... I've assumed that it is expected behaviour for the log messages at a particular level in the log hierarchy to be at a variety of log levels? So, for example, it would be perfectly acceptable to have messages at levels DEBUG and ERROR in the logger pv.sv.unhappyjob. Is that the case?

Reply:
yes